### DISCUSSION TITLE
how to replace "default" prefix from template name of view

### DISCUSSION THREAD

**Discussion by saman222** [joomengine/Joomla-Component-Builder#844] [Q&A]
hi
i want use a name for my template which begin with my own custom name, instead of default_foo.php for my view template.
for example i want add 10 different layouts (that set via $this->setlayout(bar) ) for my view of component.
but JCB create  
JOOMLA/components/com_mycomponent/views/tmpl/default_foo.php
but i want create 
JOOMLA/components/com_mycomponent/views/tmpl/bar_foo.php
how to do that?
thank you

**Comment by Llewellynvdm** [comment]
You can only do that at this point with custom file that you add via the component_files_folders area.
